pyproject.toml

```diff
@@ -69,7 +69,7 @@ Issues = "https://github.com/emdgroup/baybe/issues/"
 [project.optional-dependencies]
 chem = [
     "rdkit>=2022.3.4",
```

Comment:
see my comment about rdkit possibly being a main dep of sfkp, then this requirement would prob be obsolete

Reply:
yes, removed